### PR TITLE
Refactoring a doc for posgres cdc demo

### PR DESCRIPTION
**Pull request by gAmUssA**

### CODE REVIEW

File: postgres-debezium-ksql-elasticsearch/postgres-debezium-ksql-elasticsearch-docker-short.adoc [hunk 1]
```diff
@@ -1,15 +1,24 @@
...
+:revdate: 2019-03-22 11:14:37 -0600
+:revby: viktor at confluent dot io 
 
-This is designed to be run as a step-by-step demo. The `ksql_commands.sql` should match those run in this doc end-to-end and in theory you can just run the file, but I have not tested it. PRs welcome for a one-click script that just demonstrates the end-to-end running demo :)
+Enriching event stream data with CDC data from Postgres, stream into Elasticsearch
 
-c.10 minutes
+toc::[]
 
-*Slides: https://speakerdeck.com/rmoff/streaming-etl-in-practice-with-postgresql-apache-kafka-and-ksql[Streaming ETL in Practice with PostgreSQL, Apache Kafka, and KSQL]*
```
Comment (rmoff) [inline postgres-debezium-ksql-elasticsearch/postgres-debezium-ksql-elasticsearch-docker-short.adoc:9]: Useful to keep in, no?

**Review by rmoff — APPROVED** [review]
LGTM, although I'd prefer to keep the link in to the associated slides.

**Review by ybyzek — APPROVED** [review]
LGTM

File: postgres-debezium-ksql-elasticsearch/postgres-debezium-ksql-elasticsearch-docker-short.adoc (hunk 1 above)
Comment (gAmUssA) [reply postgres-debezium-ksql-elasticsearch/postgres-debezium-ksql-elasticsearch-docker-short.adoc:9]: Those are there. I just rewrote this line with author attributions

Comment (gAmUssA) [reply postgres-debezium-ksql-elasticsearch/postgres-debezium-ksql-elasticsearch-docker-short.adoc:9]: I moved this to line 21 

https://github.com/confluentinc/examples/pull/136/files#diff-bce90ca9f1571f481ceb433189cfd281R19

**Comment by gAmUssA** [comment]
Thank you @rmoff @ybyzek

**Pull request merged**